test/integration/servicebus/config.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+module.exports = {
+
```

Comment:
we should investigate a way to unify the integration test config files, so like protocol, host, address/target, username, password are all sort of generic? (I'm saying just a future thought, not for this PR).

The test-fixture class could eventually provide an easy means for combining these into the "config.address" variable, etc.